corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -151,7 +151,7 @@ def remove_grant(self, grantee_slug, priv_slug, dry_run=False):
         Role(slug=privileges.LOCATIONS, name='Locations', description=''),
         Role(slug=privileges.REPORT_BUILDER, name='User Configurable Report Builder', description=''),
         Role(slug=privileges.USER_CASE, name='User Case Management', description=''),
-        Role(slug=privileges.DATA_CLEANUP, name='Data Cleanup Tools',
+        Role(slug=privileges.DATA_CLEANUP, name='Data Management',
```

Comment:
oh yeah, I think the other reference below should also say `Data Management`, right?